config/Settings.ts

```diff
@@ -69,6 +63,14 @@ export const settings: Array<ISetting> = [
         i18nLabel: 'bot_username',
         required: true,
     },
+    {
+        id: AppSetting.DialogflowBotList,
+        public: true,
+        type: SettingType.CODE,
+        packageValue: `{\n    "omnichannel_bot1_name": {\n"project_id": "",\n        "client_email": "",\n        "agent_id": "",\n        "agent_region": "",\n        "agent_default_language": "",\n        "private_key": ""    }\n,    "omnichannel_bot2_name": {...}\n}`,
+        i18nLabel: 'dialogflow_bot_list_config',
```

Comment:
The label is missing in the localization file `en.json` Currently the setting label is showing `dialogflow_bot_list_config`

Reply:
Done